packages/iframe-stamper/src/index.ts

```diff
@@ -66,6 +72,34 @@ export type TIframeStamperConfig = {
   iframeContainer: HTMLElement | null | undefined;
 };
 
+export type TIframeStyles = {
+  padding?: string;
+  margin?: string;
+  borderWidth?: string;
+  borderStyle?: string;
+  borderColor?: string;
+  borderRadius?: string;
+  fontSize?: string;
+  fontWeight?: string;
+  fontFamily?: string;
+  color?: string;
+  backgroundColor?: string;
+  width?: string;
+  height?: string;
+  maxWidth?: string;
+  maxHeight?: string;
+  lineHeight?: string;
+  boxShadow?: string;
+  textAlign?: string;
+  overflowWrap?: string;
+  wordWrap?: string;
+  resize?: string;
+};
+
+export type TIframeSettings = {
+  styles?: TIframeStyles;
+};
```

Comment:
Interesting. I'm guessing you took this whitelist approach for security reasons? (as opposed to allowing arbitrary styles). If so, might be worth a comment here. I think what you have here covers a lot of use cases btw, good list!

Reply:
so far we've gotten requests for changing margin, padding, borders, and fonts which are mostly covered here. as we get more requests, we can securely add them as a strict regex pattern on frames (for security reasons). and the whitelist here in the sdk is actually more for ux than for security -- ideally a user doesn't need to constantly cross-reference the code in frames to determine which css style properties are supported